### PR TITLE
Fix theme controller to work with Result T

### PR DESCRIPTION
**Pull request by Vlad1023**
Also, commented useless unit test for themes

### CODE REVIEW

File: CharlieBackend.Api.UnitTes/Controllers/ThemesControllerTests.cs [hunk 1]
```diff
@@ -9,34 +9,33 @@
...
+		//	Assert.NotNull(themesObjectResult);
 	}
+
+	//public async Task<IList<ThemeDto>> GetThemes()
+	//{
+	//	List<ThemeDto> ThemesL = new List<ThemeDto>
+	//	{
+	//		new ThemeDto { Id = 12, Name = "Tema1" }
+	//	};
+	//	return ThemesL;
```
Comment (whereismyroot) [inline CharlieBackend.Api.UnitTes/Controllers/ThemesControllerTests.cs:39]: Do not leave commented code

**Review by whereismyroot — CHANGES_REQUESTED** [review]

File: CharlieBackend.Api/Controllers/ThemesController.cs [hunk 2]
```diff
@@ -36,7 +36,7 @@ public async Task<ActionResult<IList<ThemeDto>>> PostThemes(CreateThemeDto addTh
 
             var themeResult = await _themeService.CreateThemeAsync(addThemeModel);
 
-            return Ok(themeResult);
+            return themeResult.ToActionResult();
```
Comment (SerjKocherga) [inline CharlieBackend.Api/Controllers/ThemesController.cs:40]: Remove this empty new line.

File: CharlieBackend.Business/Services/ThemeService.cs [hunk 3]
```diff
@@ -59,26 +60,35 @@ public async Task<Result<ThemeDto>> DeleteThemeAsync(long themeId)
 
         }
 
-        public async Task<IList<ThemeDto>> GetAllThemesAsync()
+        public async Task<Result<IList<ThemeDto>>> GetAllThemesAsync()
         {
             var themes = await _unitOfWork.ThemeRepository.GetAllAsync();
```
Comment (SerjKocherga) [inline CharlieBackend.Business/Services/ThemeService.cs:67]: Remove this empty new line.

**Review by SerjKocherga — CHANGES_REQUESTED** [review]

File: CharlieBackend.Api.UnitTes/Controllers/ThemesControllerTests.cs (hunk 1 above)
Comment (Vlad1023) [reply CharlieBackend.Api.UnitTes/Controllers/ThemesControllerTests.cs]: done

File: CharlieBackend.Api/Controllers/ThemesController.cs (hunk 2 above)
Comment (Vlad1023) [reply CharlieBackend.Api/Controllers/ThemesController.cs]: done

File: CharlieBackend.Business/Services/ThemeService.cs (hunk 3 above)
Comment (Vlad1023) [reply CharlieBackend.Business/Services/ThemeService.cs]: done

**Review by sober607 — APPROVED** [review]
Ok

**Review by whereismyroot — APPROVED** [review]

**Review by SnipeFrost — APPROVED** [review]

**Review by BohdanBBG — APPROVED** [review]

**Review by SerjKocherga — APPROVED** [review]

**Pull request merged**